etcd/fs/fs_test.go

```diff
@@ -39,13 +43,45 @@ const (
 	superblock = "/some/superblock" // TODO: generate randomly per test?
 )
 
+// TestMain will be called instead of running the tests directly. This allows
+// us to start mgmt before running the tests, ensuring there will be an etcd
+// node available to use as a filesystem backend.
+func TestMain(m *testing.M) {
+	// run mgmt as etcd backend
+	cmdName, err := integration.BinaryPath()
+	if err != nil {
+		fmt.Printf("error getting binary path: %s", err)
+		os.Exit(1)
+	}
+	cmd := exec.Command(cmdName, "run", "--tmp-prefix")
+	if err := cmd.Start(); err != nil {
+		fmt.Printf("error starting command %v: %s", cmd, err)
+		os.Exit(1)
+	}
+
+	// run the tests
+	exit := m.Run()
+
+	// cleanup when we're done
+	if err := cmd.Process.Signal(syscall.SIGQUIT); err != nil {
+		fmt.Printf("error sending quit signal: %s", err)
```

Comment:
printf needs a \n and err is usually %v IIRC
Too bad there's no proper log method for `m`